access/src/main/resources/recordTransformations/modsToFullRecord.xsl

```diff
@@ -525,6 +525,11 @@
 								<xsl:when test="local-name() = 'cartographics'">
 									<xsl:call-template name="cartographics"/>
 								</xsl:when>
+								<xsl:when test="starts-with(../@displayLabel, 'RLA') and ../@xlink:href">
```

Comment:
this doesn't need to be RLA specific, any xlink:href in a subject should be fine to link to